{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "total_count": 3,
    "last_updated": "2025-05-13T13:54:55.775432",
    "severity_distribution": {
      "critical": 0,
      "high": 1,
      "medium": 0,
      "low": 1,
      "none": 1
    }
  },
  "cves": [
    {
      "id": "CVE-2025-22249",
      "publishedDate": "2025-05-13T05:08:03.265Z",
      "lastModifiedDate": "2025-05-13T13:49:59.998Z",
      "description": "VMware Aria automation contains a DOM based Cross-Site Scripting (XSS) vulnerability. A malicious actor may exploit this issue to steal the access token of a logged in user of VMware Aria automation appliance by tricking the user into clicking a malicious crafted payload URL.",
      "severity": 8.2,
      "references": [
        {
          "url": "https://support.broadcom.com/web/ecx/support-content-notification/-/external/content/SecurityAdvisories/0/25711",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "defaultStatus": "unaffected",
          "platforms": [
            "any"
          ],
          "product": "Vmware Aria Automation",
          "vendor": "VMware",
          "versions": [
            {
              "lessThan": "8.18.1 patch2",
              "status": "affected",
              "version": "8.18.x",
              "versionType": "custom"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "platforms": [
            "any"
          ],
          "product": "VMware Cloud Foundation",
          "vendor": "VMware",
          "versions": [
            {
              "lessThan": "8.18.1 patch 2",
              "status": "affected",
              "version": "5.x",
              "versionType": "custom"
            },
            {
              "lessThan": "8.18.1 patch 2",
              "status": "affected",
              "version": "4.x",
              "versionType": "custom"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "platforms": [
            "any"
          ],
          "product": "VMware Telco Cloud Platform",
          "vendor": "VMware",
          "versions": [
            {
              "lessThan": "8.18.1 patch 2",
              "status": "affected",
              "version": "5.x",
              "versionType": "custom"
            }
          ]
        }
      ],
      "problemType": [],
      "fix_suggestion": "无法生成修复建议"
    },
    {
      "id": "CVE-2025-22246",
      "publishedDate": "2025-05-13T05:14:40.968Z",
      "lastModifiedDate": "2025-05-13T13:49:09.193Z",
      "description": "Cloud Foundry UAA release versions from v77.21.0 to v7.31.0 are vulnerable to a private key exposure in logs.",
      "severity": 3,
      "references": [
        {
          "url": "https://www.cloudfoundry.org/blog/cve-2025-22246-uaa-private-key-exposure/",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "defaultStatus": "unaffected",
          "platforms": [
            "Any"
          ],
          "product": "UAA",
          "vendor": "Cloud Foundry",
          "versions": [
            {
              "lessThan": "v77.32.0",
              "status": "affected",
              "version": "v77.21.0",
              "versionType": "git"
            }
          ]
        },
        {
          "defaultStatus": "unaffected",
          "platforms": [
            "any",
            "v45.1.0"
          ],
          "product": "CF deployment",
          "vendor": "Cloud Foundry",
          "versions": [
            {
              "changes": [
                {
                  "at": "v49.0.0",
                  "status": "unaffected"
                }
              ],
              "lessThanOrEqual": "v48.11.0",
              "status": "affected",
              "version": "v45.1.0",
              "versionType": "git"
            }
          ]
        }
      ],
      "problemType": [],
      "fix_suggestion": "无法生成修复建议"
    },
    {
      "id": "CVE-2025-30453",
      "publishedDate": "2025-05-12T21:42:20.267Z",
      "lastModifiedDate": "2025-05-13T13:40:33.488Z",
      "description": "The issue was addressed with additional permissions checks. This issue is fixed in macOS Sequoia 15.4, macOS Ventura 13.7.6, macOS Sonoma 14.7.6. A malicious app may be able to gain root privileges.",
      "severity": "N/A",
      "references": [
        {
          "url": "https://support.apple.com/en-us/122717",
          "type": "reference"
        },
        {
          "url": "https://support.apple.com/en-us/122718",
          "type": "reference"
        },
        {
          "url": "https://support.apple.com/en-us/122373",
          "type": "reference"
        }
      ],
      "affected": [
        {
          "vendor": "Apple",
          "product": "macOS",
          "versions": [
            {
              "version": "unspecified",
              "status": "affected",
              "lessThan": "14.7",
              "versionType": "custom"
            }
          ]
        },
        {
          "vendor": "Apple",
          "product": "macOS",
          "versions": [
            {
              "version": "unspecified",
              "status": "affected",
              "lessThan": "13.7",
              "versionType": "custom"
            }
          ]
        },
        {
          "vendor": "Apple",
          "product": "macOS",
          "versions": [
            {
              "version": "unspecified",
              "status": "affected",
              "lessThan": "15.4",
              "versionType": "custom"
            }
          ]
        }
      ],
      "problemType": [
        "A malicious app may be able to gain root privileges"
      ],
      "fix_suggestion": "无法生成修复建议"
    }
  ]
}